src/main/resources/application.conf

```diff
@@ -35,7 +35,7 @@ spray.can {
 
 backend {
   // Either "jes", "local", or "sge" (case insensitive)
-  backend = "sge"
+  backend = "local"
```

Comment:
I twiddled this before I commented on your PR. Either way this needs to be local once all is said and done but not a huge deal while you're messing about.

I think it'd be good for everyone to get in the habit of using their own personal conf files instead of twiddling this one though.